Misc/NEWS.d/next/Library/2019-07-05-20-54-46.bpo-35711.xYuDRM.rst

```diff
@@ -0,0 +1 @@
+Adding new :data:`mmap.MAP_CONCEAL` OpenBSD constant.
```

Comment:
Can you try to merge the 2 changelog entries? Put content of the 2nd entry in the first one, remove the second file.